### ISSUE TITLE
Download Station Synology

### ISSUE THREAD

**Issue by nightmare04** [werwolfby/monitorrent#235]
Возможно добавить поддержку встроенного клиента от Synology DSM?
Просто недавно наткнулся на расширение для браузера https://www.download-station-extension.com/ которое неплохо с этим справляется, а так как я думаю что я тут не один который поставил Monitorrent на DSM думаю для многих это будет актуально.

**Comment by moonkir** [comment]
В приципе можно скачивать торрент файл мониторрентом в папку - а эту папку скормить менеджеру закачек в DSM

**Comment by nightmare04** [comment]
Согласен, но тогда пропадает возможность скачивать в определенную папку... На сколько мне известно, там тот же transmission, так что это должно быть возможно.

**Comment by c4uran** [comment]
а есть инструкция как поставить на dsm? я открыл докер, там скачал и запустил контейнер, пытаюсь обратится к нему через http://server:6687 на что получаю что не может открыть страницу...

**Comment by moonkir** [comment]
В настройках контейнера, выбери какой порт использует приложение и какой пробрасывать наружу, например 6687 оба
Добавь расположение базы, у меня это dsm/docker/monitorrent/monitorrent.db и путь где мониторрент ищет базу /var/www/monitorrent/monitorrent.db

**Comment by c4uran** [comment]
А можно пожалуйста подробней как это сделать?

**Comment by moonkir** [comment]
Останавливаешь работу контейнера, заходишь в его настройки
На вкладке настройки портов, локальный порт и порт контейнера пишешь 6687
на вкладке том  - добавляешь файл - левая панель (заранее создай его, у меня он по пути dsm/docker/monitorrent/monitorrent.db) а в правую часть пишешь /var/www/monitorrent/monitorrent.db
Запускаешь контейнер - и должно работать
